pootle/templates/browser/index.html

```diff
@@ -143,10 +143,16 @@ <h3 class="top">{% trans "Updates" %}</h3>
 {% endif %}
 
 <div id="autorefresh-notice">
-  {% blocktrans trimmed %}
-  Some data on this page is currently being calculated, and the page will be
-  refreshed automatically in <strong>30 seconds</strong>.
-  {% endblocktrans %}
+  {% if stats_banner_auto_refresh_enabled %}
+    {% blocktrans trimmed %}
+    Some data on this page is currently being calculated, and the page will be
+    refreshed automatically in <strong>30 seconds</strong>.
+    {% endblocktrans %}
+  {% elif has_site_admin_permission %}
+    {% trans "Stats are out of date. See documentation for dealing with stats issues." %}
+  {% else %}
+    {% trans "The stats on this page may be out of date." %}
+  {% endif %}
```

Comment:
@unho it will disable counter because it relies on `<strong/>` element. So I'll leave the current order.